src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -223,6 +223,11 @@ protected void processFiltered(File aFile, List<String> aLines)
                     getId(),
                     this.getClass(), null));
         }
+        catch (final NullPointerException npe) {
+            System.err.println("Exception happened during analyzing: " + aFile.getPath()
```

Comment:
thanks, updated message and removed odd throwing of NPEs.
About 'if ```aFile``` is currently null' - I think current condition in 'catch' block covers such possible case